app/controllers/home_controller.rb

```diff
@@ -5,7 +5,23 @@ def type;end
 
   def faqs; end
 
-  def profile_psychologist; end
+  def profile_psychologist; 
+    
+  end
+
+  def psychologist_List
```

Comment:
In ruby, usually, method names are always lowercase. Maybe you'd like to change to this to `psychologist_list`?